src/Display/Progress/Progress.tsx

```diff
@@ -23,10 +23,12 @@ const Progress: React.FunctionComponent<Props> = (props) => {
 
   let normalizedPercentage;
   let normalizedSize;
-
-  if (percentage < 0 || percentage > 100) {
+  if (isNumber(percentage) && percentage < 0 || percentage > 100) {
     normalizedPercentage = percentage < 0 ? 0 : percentage > 100 ? 100 : percentage;
-    console.warn(`Invalid prop value on Progress component: percentage prop expected a value between 0-100. Received ${percentage} instead.`)
+    console.warn(`Invalid prop value on Progress component: percentage prop expected a number between 0-100. Received ${percentage} instead.`)
```

Comment:
@franciscrispin warning should also inform the client what you resolved the final number to.